api/src/notification/templates.ts

```diff
@@ -19,9 +11,9 @@ export type NotificationType =
 
 export const FONT_SETUP = `<style>@import url('https://fonts.cdnfonts.com/css/courier-prime');p {font-family: 'Courier Prime', sans-serif;}</style>`;
 
-export const singup = (recipients: Recipients) => ({
+export const singup = (recipients: RecipientsDto): EmailDto => ({
   from: 'CineSync <mail@cinesync.me>',
-  to: recipients.userEmail,
+  to: recipients.userEmail || '',
```

Comment:
`recipients.userEmail || ''`? Do we want to fire an email with no recipient? same for listShare, newComment, accountDeleted.

Maybe I'm missing something here.